pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -174688,7 +174688,6 @@ self: {
        description = "A Pandoc filter for emphasizing code in fenced blocks";
        license = stdenv.lib.licenses.mpl20;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
You'll also need to delete the line, otherwise hydra won't build this package.

```suggestion
```

Reply:
Ah, of course. Fixed now, thanks!